typescript/infra/scripts/check/check-utils.ts

```diff
@@ -42,33 +39,36 @@ import {
   withFork,
   withGovern,
   withModule,
+  withPushMetrics,
   withWarpRouteId,
 } from '../agent-utils.js';
 import { getEnvironmentConfig, getHyperlaneCore } from '../core-utils.js';
 import { getHelloWorldApp } from '../helloworld/utils.js';
 
-export function getCheckArgs() {
+export function getCheckBaseArgs() {
   return withAsDeployer(
     withGovern(withChain(withFork(withContext(getRootArgs())))),
   );
 }
 
-export function getCheckDeployArgs() {
-  return withWarpRouteId(withModule(getCheckArgs()));
+export function getCheckWarpDeployArgs() {
+  return withPushMetrics(getCheckBaseArgs());
 }
 
-export async function check(argv?: Record<string, any>) {
-  const {
-    fork,
-    govern,
-    module,
-    environment,
-    context,
-    chain,
-    asDeployer,
-    warpRouteId,
-  } = argv ?? (await getCheckDeployArgs().argv);
+export function getCheckDeployArgs() {
+  return withWarpRouteId(withModule(getCheckBaseArgs()));
+}
 
+export const getGovernor = async (
```

Comment:
nit: any reason to make this an arrow function instead of a normal `async function`? I don't feel strongly but just for consistency in infra we've usually had top level fns like this stay using the `function` keyword

Reply:
No reason, I'll change